applications/asset_tracker_v2/src/ext_sensors/ext_sensors_bsec.c

```diff
@@ -51,6 +80,8 @@ float temp_offset = (CONFIG_EXTERNAL_SENSORS_BSEC_TEMPERATURE_OFFSET / (float)10
 #define BSEC_STACK_SIZE CONFIG_EXTERNAL_SENSORS_BSEC_THREAD_STACK_SIZE
 K_THREAD_STACK_DEFINE(thread_stack, BSEC_STACK_SIZE);
 
+K_TIMER_DEFINE(bsec_save_state_timer, NULL, NULL);
```

Comment:
A comment on what this variable is used for would be helpful.

Reply:
done